### DISCUSSION TITLE
Manual/ManagedCodeStripping

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#207] [Notes]
# Manual/ManagedCodeStripping

https://docs.unity3d.com/Manual/ManagedCodeStripping.html

**Comment by mvi** [comment]
By default specifying an assembly in link.xml will require it to be present at link stage otherwise you may get an error.  
You can mark an assembly as optional by adding the following attribute: `ignoreIfMissing="1"`  
for example  
  

```xml
<assembly fullname="UnityEngine.Analytics" ignoreIfMissing="1">
```
